### ISSUE TITLE
env-preset adds more transforms than required

### ISSUE THREAD

**Issue by rosenfeld** [babel/babel#6968]
Choose one: is this a bug report or feature request? *Bug*

### Input Code

```js
class Sample {
  constructor() {
    const a = () => 'a'
  }
}
```

### Babel/Babylon Configuration (.babelrc, package.json, cli command)

```js
{
  "presets": [
    [
      "env", {
/*        "exclude": [ "transform-es2015-arrow-functions", "transform-es2015-classes" ],*/
        "modules": false,
        "targets": { "browsers": "Chrome >= 62" }
      }
    ]
  ]
}
```

### Expected Behavior

Since Chrome 62 supports arrow functions and classes, I'd expect those not to be converted to function, but it doesn't happen unless I explicitly remove those transformations in the "exclude" option.

### Current Behavior

Arrow keys and classes are converted to functions even when not required by all supported browsers.

### Context

I'm creating two bundles. One for modern browsers and another for older ones. I want modern browsers to download as little code as possible for performance reasons, so I want to keep the newer syntax whenever they are supported natively. Those new features might be well optimized by browsers too.

### Your Environment

| software         | version(s)
| ---------------- | -------
| Babel            |  6.24.1
| node             | v6.10.3
| npm              | 5.5.1
| Operating System | Debian sid Linux

**Comment by existentialism** [comment]
@rosenfeld can you set `debug: true` and show us the output? Also, can you do a `npm ls browserslist`? One possibility is that the version of browserslist installed doesn't recognize Chrome 62.

**Comment by rosenfeld** [comment]
Wow, this was weird. I must have missed something. I ran `npm ls browserslist`, enabled debug and suddenly it worked. Then I removed `debug: true` and it kept working, so right now I can't reproduce it. It's probably something else I've done recently in the project but since there's a lot of time since my last commit I won't be able to understand what happened. I'll close for now and if it happens again I'll reopen it. Sorry for bothering.
